charts/generic-service/templates/controller.yaml

```
@@ -400,6 +400,8 @@ spec:
             - name: JAEGER_SERVICE_NAME
               value: {{ include "generic-service.fullname" . }}
             {{- end }}
+            - name: OTEL_EXPORTER_OTLP_INSECURE
+              value: {{ .Values.tracing.insecure }}
```

Booleans need to be quoted when used in environment variables, since only strings are allowed.

```suggestion
              value: {{ .Values.tracing.insecure | quote }}
```

---

Done